docs/AppBar.md

```diff
@@ -0,0 +1,58 @@
+# App Bar
+
+The `<AppBar>` component is an extension of the default AppBar from Material UI that can be resized / collapsed as the page is scrolled. It supports three modes:
```

Comment:
"it supports three modes:" is followed by 2 pictures and then the usage section below the pictures.  It's a little weird reading this because it's not explicit what the 3 modes are.

Reply:
Yeah I'll reorganize this a bit....it doesn't look weird when you're just looking at the markdown file 😛

Reply:
Oh, I definitely wrote descriptions at one point...I think they got lost on the day when I accidentally deleted the local file before saving it after moving it into the components library.